### PR TITLE
Image cropper connected instead of focal point

### PR DESCRIPTION
**Pull request by Flight**
Image cropper connected, focal point removed.
Will need BE help here. Can't merge into master, because it will fully break the detailed page.

### CODE REVIEW

File: filer/templates/admin/filer/base_site.html [hunk 1]
```diff
@@ -18,15 +18,16 @@
...
+    <script src="{% static 'filer/js/libs/jquery.cookie.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/fileuploader.min.js' %}"></script>
     <script src="{% static 'admin/js/admin/RelatedObjectLookups.js' %}"></script>
     <script src="{% static 'filer/js/addons/popup_handling.js' %}"></script>
     <script src="{% static 'filer/js/addons/table-dropzone.js' %}"></script>
     <script src="{% static 'filer/js/addons/upload-button.js' %}"></script>
-
-    <script src="{% static 'filer/js/addons/mediator.min.js' %}"></script>
-    <script src="{% static 'filer/js/addons/focal-point.js' %}"></script>
+    <!-- <script src="{% static 'filer/js/addons/focal-point.js' %}"></script> -->
```
Comment (vxsx) [inline filer/templates/admin/filer/base_site.html]: should remove

**Comment by vxsx** [comment]
overall LGTM

**Comment by Flight** [comment]
Will need BE help here. Can't merge into master, because it will fully break the detailed page.

**Comment by Flight** [comment]
Closed because of github issues

**Pull request closed without merging**